Client/loader/Main.cpp

```diff
@@ -78,7 +78,9 @@ MTAEXPORT int DoWinMain(HINSTANCE hLauncherInstance, HINSTANCE hPrevInstance, LP
 
     // Stuff
     HandleCustomStartMessage();
-    ForbodenProgramsMessage();
+    #if !defined(MTA_DEBUG)

```

Comment:
```suggestion
    #ifndef MTA_DEBUG
```

I think `ifdef`/`ifndef` is preferred (where possible)